woodwork/type_sys/type_system.py

```diff
@@ -93,7 +93,7 @@
 
 DEFAULT_TYPE = Unknown
 
-INFERENCE_SAMPLE_SIZE = 100000
+INFERENCE_SAMPLE_SIZE = 10_000
```

Comment:
Shouldn't this be an integer with no underscore? 
```suggestion
INFERENCE_SAMPLE_SIZE = 10000
```

Reply:
Decided to follow [PEP 515](https://peps.python.org/pep-0515/) for this since it makes it easier to read